### DISCUSSION TITLE
Welcome to TikTok-Bot Discussions!

### DISCUSSION THREAD

**Discussion by ReverseAndCode** [ReverseAndCode/TikTok-Bot#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by jessicaross12** [comment]
Hello new to coding and Github. I found the Tik Tok bot interesting. Can you describe to me what it'll do?

**Reply by ReverseAndCode** [reply]
Hi!

Thank you for your reply.

The TikTok bot (still under development) will offer you everything you can't do at the TikTok website, e. g. mass-scraping the followers of other TikTok users, mass-follow the users, mass-message the users, mass-upload videos. So basically all features of the official TikTok iOS app, but totally automated.

You will be able to use hundreds of your TikTok accounts at the same time, with scheduled activities running 24/7/365 in the background.  

This screenshot shows the "Follower" module in action:

![TikTok Bot - Follower - 1 - 1408 x 741 - PNG](https://user-images.githubusercontent.com/74844939/102999671-c241cd00-4529-11eb-9ff5-2eaaaa82ca13.png)

**Comment by xiaohai1023** [comment]
找一个tiktok的协议作者
